pkg/volume/csi/csi_client.go

```diff
@@ -93,10 +93,12 @@ type csiDriverName string
 
 // csiClient encapsulates all csi-plugin methods
 type csiDriverClient struct {
-	driverName          csiDriverName
-	addr                csiAddr
-	nodeV1ClientCreator nodeV1ClientCreator
-	nodeV0ClientCreator nodeV0ClientCreator
+	driverName                csiDriverName
+	addr                      csiAddr
+	nodeV1ClientCreator       nodeV1ClientCreator
+	nodeV0ClientCreator       nodeV0ClientCreator
+	controllerV1ClientCreator controllerV1ClientCreator
+	controllerV0ClientCreator controllerV0ClientCreator
```

Comment:
I don't think you need a `controllerV0ClientCreator` since the client did not use controller calls prior to `V1`.